modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/utils/constants.ts

```diff
@@ -37,4 +37,15 @@ export const RETRIEVE_TYPES = {
 
 export const STR_BLANK = '';
 
-export const SYSTEM_OBJECTS_WHITELIST = ['L_ACCOUNT', 'L_USER'];
+const MODIFIABLE_SYSTEM_OBJECTS = [
+	'L_API_SORT',
+	'L_API_FILTER',
+	'L_API_ENDPOINT',
+	'L_API_PROPERTY',
+	'L_API_SCHEMA',
+	'L_API_APPLICATION',
+];
+
+export const SYSTEM_OBJECTS_WHITELIST = Liferay.FeatureFlags['LPD-21414']
```

Comment:
I would call it `UNMODIFIABLE_OBJECTS_WHITELIST` just to follow the convention naming